mopidy/mpd/protocol/playback.py

```diff
@@ -426,3 +426,27 @@ def stop(context):
         Stops playing.
     """
     context.core.playback.stop()
+
+
+@protocol.commands.add('volume', change=protocol.INT)
+def volume(context, change):
+    """
+    *musicpd.org, playback section:*
+
+        ``volume {CHANGE}``
+
+        Changes volume by amount ``CHANGE``.
+
+        Note: ``volume`` is deprecated, use ``setvol`` instead.
+    """
+    if change < -100 or change > 100:
+        raise exceptions.MpdArgError('Invalid volume value', command='volume')
```

Comment:
Aha, I was just wondering about the `MpdSystemError` not having `command=` and the tests still passing.